src/python/WMQuality/Emulators/DBSClient/DBSReader.py

```diff
@@ -9,19 +9,19 @@
 
 class _MockDBSApi():
     """Mock dbs api"""
-    def __init__(self, args):
```

Comment:
I don't see how any of the changes here relate to the other changes or to any issue.